src/alert-manager/deploy/alert-configmap.yaml.template

```diff
@@ -28,4 +35,15 @@ data:
           html: '{{ "{{" }} template "email.pai.html" . {{ "}}" }}'
           headers:
             subject: '{{ cluster_cfg["cluster"]["common"]["cluster-id"] }}: {{ "{{" }} template "__subject" . {{ "}}" }}'
+    - name: "pai-alert-handler"
```

Comment:
the name `pai-alert-handler` is too similar to `pai-alert`, could you please replace it by something like `email_then_stop_handler`?

Reply:
Fixed